dashboard/db/migrate/20160805190505_drop_user_trophies_and_trophies.rb

```diff
@@ -0,0 +1,6 @@
+class DropUserTrophiesAndTrophies < ActiveRecord::Migration
+  def up
```

Comment:
Yes, in that `rake db:migrate && rake db:rollback && rake db:migrate` will execute without error (since the table is being conditionally dropped).

No, in that the table is not being recreated (given that dropping the table purges the data). That said, we do have `mysqldump`s of the tables (including definition and data) on S3.